style.css

```diff
@@ -461,3 +453,182 @@ form {
   color: #5e6c84;
   border-color: #5e6c84;
 }
+
+@media (min-width: 768px) {
+  .first_project,
+  .second_project,
+  .third_project,
+  .forth_project {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border: 2px solid grey;
+    border-radius: 12px;
+    padding: 15px 35px;
+    background-color: #fff;
+    gap: 50px;
+    margin-bottom: 20px;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .navbar {
+    width: 100%;
+    background-color: white;
+    padding: 30px 154px;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    justify-content: space-between;
+    gap: 850px;
+  }
+
+  .dropdown {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 100px;
+    padding-bottom: 20px;
+    margin-top: 20px;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .Tonic {
+    width: 600px;
+    height: 400px;
+  }
+
+  .flex_list {
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+  }
+
+  .project_description {
+    width: 100%;
+    height: fit-content;
+    margin-bottom: 10px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .project_name {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    margin-bottom: 20px;
+  }
+
+  .About_Section {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-around;
+    padding: 100px 120px;
+  }
+
+  .About_Section_description {
+    width: 400px;
+  }
```

Comment:
- [ ] [Optional] As the project requirement please stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). See image for reference. please fix the border as well and try to fix in every section.
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199728329-df347b2e-7722-4736-b0a8-085d623d54e5.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/199728684-5338366f-a21b-414d-bc99-fb8bac59f127.png)